rfc-020-use-rollbar-to-monitor-rails-errors.md

```diff
@@ -0,0 +1,26 @@
+# Use Rollbar to monitor application errors for Rails projects
+
+## Summary
+
+Use Rollbar as the service to monitor and manage application errors for Rails
+ projects.
+
+## Problem
+
+We need to use a tool that collects errors that happen across our live
+ environments.
+We need a tool that can notify the team of new or important errors.
+We need a way to record and share the outputs of our investigations and any
+ plans to tackle the problem, more investigation, intended fixes etc.
+We need a way to group common issues.
+
+## Proposal
+
+We SHOULD use Rollbar as our default tool for all Rails projects.
+We MAY use another service if the client has a preference, this MUST be captured
+ in an ADR within the project itself.
+
+## Next steps
+
+Add Rollbar to the rails-template repository
+ <https://github.com/dxw/rails-template/>
```

Comment:
Can we make this a link instead of putting the URL?

Reply:
I just did `npm lint:fix` and trusted that it would sort it all 🤷‍♂

Reply:
Linter doesn't know what your intentions are :).